code/core/arm9/gbarunner9.ld

```diff
@@ -243,6 +244,15 @@ SECTIONS
 		__itcm_end = ABSOLUTE(.);
 	} >itcm AT>vramab :itcm = 0xff
 
+    .ewram :
```

Comment:
Ensure the identation is identical to the rest of the file. It seems this file is using tabs for some reason